### PR TITLE
i18n(zh-cn): Translate 'sidebar' and `features/*`

### PR DESCRIPTION
**Pull request by GuoJikun**
#### What kind of changes does this PR include?

- Translated content

#### Description

- [x]  'sidebar' in 'astro.config.mjs'
- [x]  `features/persisted-scope.mdx`
- [x]  `features/positioner.mdx`
- [x]  `features/upload.mdx`
- [x]  `features/websocket.mdx`
- [x]  `features/stronghold.mdx`
- [x]  `features/shell.mdx`
- [x]  `features/sql.mdx`
- [x]  `features/file-system.mdx`
- [x] `features/commands.mdx`
- [x] `features/window-customization.mdx`
- [x] `features/single-instance.mdx`
- [x] `features/store.mdx`
- [x] Small optimizations

### CODE REVIEW

**Comment by GuoJikun** [comment]
@dklassic This PR need review if you've got some time.

File: src/content/docs/zh-cn/features/persisted-scope.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: 持久化作用域(Persisted Scope)
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/persisted-scope.mdx]:
```suggestion
title: 持久化作用域（Persisted Scope）
```

File: src/content/docs/zh-cn/features/positioner.mdx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    		npm="npm run tauri add positioner"
+    		yarn="yarn run tauri add positioner"
+    		pnpm="pnpm tauri add positioner"
+    		cargo="cargo tauri add positioner"
+    	/>
+
+    </TabItem>
+    <TabItem label = "手动">
+    	<Steps>
+    	1. 在你的 `Cargo.toml` 文件中添加以下内容来安装 positioner 插件。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/positioner.mdx]:
```suggestion
    	1. 在你的 `Cargo.toml` 文件中添加以下内容来安装定位器插件。
```

File: src/content/docs/zh-cn/features/positioner.mdx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+				})
+				.run(tauri::generate_context!())
+				.expect("error while running tauri application");
+		}
+		```
+</Steps>
+
+## 用法
+
+插件的 api 可以通过 JavaScript guest 绑定获得。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/positioner.mdx]:
```suggestion
插件的 API 可以通过 JavaScript Guest 绑定获得。
```

File: src/content/docs/zh-cn/features/shell.mdx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    			pnpm = "pnpm add @tauri-apps/plugin-shell"
+    		/>
+    	</Steps>
+    </TabItem>
+
+</Tabs>
+
+## 用法
+
+shell 插件有 JavaScript 和 Rust 两种版本。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/shell.mdx]:
```suggestion
Shell 插件有 JavaScript 和 Rust 两种版本。
```

File: src/content/docs/zh-cn/features/sql.mdx [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+```rust title="src-tauri/src/main.rs" ins={3}
+fn main() {
+    tauri::Builder::default()
+        .plugin(tauri_plugin_sql::Builder::default().build())
+        .run(tauri::generate_context!())
+        .expect("error while running tauri application");
+}
+```
+
+之后，所有插件的 api 都可以通过 JavaScript Guest 绑定使用。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/sql.mdx]:
```suggestion
之后，所有插件的 API 都可以通过 JavaScript Guest 绑定使用。
```

File: src/content/docs/zh-cn/features/sql.mdx [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+  </TabItem>
+</Tabs>
+
+## 迁移
+
+这个插件支持数据库迁移，允许你管理数据库模式随时间的变化。
+
+### 定义迁移
+
+迁移在Rust中使用 [`Migration`](https://docs.rs/tauri-plugin-sql/latest/tauri_plugin_sql/struct.Migration.html) 结构体定义。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/sql.mdx]:
```suggestion
迁移在 Rust 中使用 [`Migration`](https://docs.rs/tauri-plugin-sql/latest/tauri_plugin_sql/struct.Migration.html) 结构体定义。
```

File: src/content/docs/zh-cn/features/sql.mdx [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+每个迁移都应该包含唯一的版本号、描述、要执行的 SQL 和迁移类型(向上或向下)。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/sql.mdx]:
```suggestion
每个迁移都应该包含唯一的版本号、描述、要执行的 SQL 和迁移类型（向上或向下）。
```

File: src/content/docs/zh-cn/features/sql.mdx [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+                .add_migrations("sqlite:mydatabase.db", migrations)
+                .build(),
+        )
+        ...
+}
+```
+
+### 应用迁移
+
+迁移在插件初始化时自动应用。插件针对连接字符串指定的数据库运行这些迁移。确保迁移按照正确的顺序定义，并且是幂等的(可以安全运行多次)。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/sql.mdx]:
```suggestion
迁移在插件初始化时自动应用。插件针对连接字符串指定的数据库运行这些迁移。确保迁移按照正确的顺序定义，并且是幂等的（可以安全运行多次）。
```

File: src/content/docs/zh-cn/features/sql.mdx [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+
+### 迁移管理
+
+- **版本控制**: 每个迁移必须有一个唯一的版本号。这对于确保迁移按正确的顺序应用至关重要。
+- **幂等性**: 编写迁移时，要确保它们能够安全重新运行，而不会导致错误或意外后果。
+- **测试**: 彻底测试迁移，确保它们按预期工作，并且不会损害数据库的完整性。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/sql.mdx]:
```suggestion
- **版本控制**：每个迁移必须有一个唯一的版本号。这对于确保迁移按正确的顺序应用至关重要。
- **幂等性**：编写迁移时，要确保它们能够安全重新运行，而不会导致错误或意外后果。
- **测试**：彻底测试迁移，确保它们按预期工作，并且不会损害数据库的完整性。
```

File: src/content/docs/zh-cn/features/stronghold.mdx [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+## 用法
+
+### 使用自定义密码散列函数初始化
+
+```rust title="src-tauri/src/lib.rs"
+pub fn run() {
+		tauri::Builder::default()
+				.plugin(
+						tauri_plugin_stronghold::Builder::new(|password| {
+								// 在这里使用 argon2, blake2b 或任何其他安全算法对密码进行散列。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/stronghold.mdx]:
```suggestion
								// 在这里使用 argon2、blake2b 或任何其他安全算法对密码进行散列。
```

Comment (dklassic) [reply src/content/docs/zh-cn/features/stronghold.mdx]: Might want to be careful committing this on GitHub as it tends to break the formatting.

File: src/content/docs/zh-cn/features/stronghold.mdx [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+						Ok(())
+				})
+				.run(tauri::generate_context!())
+				.expect("error while running tauri application");
+}
+```
+
+### 在 JavaScript 使用
+
+stronghold 插件可以在 JavaScript 中使用。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/stronghold.mdx]:
```suggestion
Stronghold 插件可以在 JavaScript 中使用。
```

File: src/content/docs/zh-cn/features/upload.mdx [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+    		/>
+
+    	</Steps>
+    </TabItem>
+
+</Tabs>
+
+## 用法
+
+一旦完成了插件的注册和设置过程，就可以通过 JavaScript Guest 绑定访问它的所有 api。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/upload.mdx]:
```suggestion
一旦完成了插件的注册和设置过程，就可以通过 JavaScript Guest 绑定访问它的所有 API。
```

File: src/content/docs/zh-cn/features/websocket.mdx [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+    		/>
+
+    	</Steps>
+    </TabItem>
+
+</Tabs>
+
+## 用法
+
+websocket 插件可以在 JavaScript 中使用。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/websocket.mdx]:
```suggestion
WebSocket 插件可以在 JavaScript 中使用。
```

**Review by dklassic — CHANGES_REQUESTED** [review]

**Comment by GuoJikun** [comment]
Should "references" be translated as' 参考' or '引用'

image

File: astro.config.mjs [hunk 14]
```diff
@@ -99,104 +99,172 @@ export default defineConfig({
...
+								'zh-CN': "故障排除"
+							},
 							link: 'guides/troubleshoot',
 						},
 					],
 				},
 				{
 					label: 'Guides',
+					translations:{
+						'zh-CN': "指导"
```
Comment (dklassic) [inline astro.config.mjs]:
```suggestion
						'zh-CN': "指引"
```
Or maybe '指南'.
As '指导' is more verb and the guide here is supposed to be a noun.

File: astro.config.mjs [hunk 15]
```diff
@@ -99,104 +99,172 @@ export default defineConfig({
...
+								'zh-CN': "插件开发"
+							},
 							link: 'guides/plugins',
 						},
 					],
 				},
 				{
 					label: 'References',
+					translations: {
+						"zh-CN": "引用"
```
Comment (dklassic) [inline astro.config.mjs]:
```suggestion
						"zh-CN": "参考"
```
Or '参考资料'.

File: astro.config.mjs [hunk 16]
```diff
@@ -99,104 +99,172 @@ export default defineConfig({
...
+					},
 					collapsed: true,
 					items: [
 						{
 							label: 'List of References',
+							translations: {
+								"zh-CN": "引用列表"
```
Comment (dklassic) [inline astro.config.mjs]:
```suggestion
								"zh-CN": "参考列表"
```
Or '参考资料列表'.

**Review by dklassic — CHANGES_REQUESTED** [review]
> Should "references" be translated as' 参考' or '引用'

Now that you mentioned it, '参考' is indeed a better word than '引用' in this case, as it is supposed to be reference materials. I'm thinking maybe we should go one step further to mark it as '参考资料'.

On the other hand we should probably use '指引' or '指南' for 'guide' as it is more of a noun here and '指导' is more of a verb.

File: astro.config.mjs (hunk 14 above)
Comment (GuoJikun) [reply astro.config.mjs]: 指南

**Comment by GuoJikun** [comment]
Today I translated four more pages, do I need to commit? @dklassic

**Comment by dklassic** [comment]
> Today I translated four more pages, do I need to commit? @dklassic

Okay! I think I have time to go through it :)

**Comment by GuoJikun** [comment]
Have committed. @dklassic

File: src/content/docs/zh-cn/features/commands.mdx [hunk 17]
```diff
@@ -0,0 +1,390 @@
+---
+title: 从前端调用 Rust
+---
+
+Tauri 提供了一个简单而强大的 `command` 系统，用于从你的 Web 应用程序中调用 Rust 函数。
+命令可以接受参数和返回值。它们也可以返回错误并且是 `async` 的。
+
+## 基本的例子
+
+命令在 `src-tauri/src/lib.rs` 文件中定义。要创建命令，只需添加一个函数并用 `#[tauri::command]` 注释它。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
命令在 `src-tauri/src/lib.rs` 文件中定义。要创建命令，只需添加一个函数并用 `#[tauri::command]` 标注它。
```
Though technically correct, '注释' can be easily confused with commenting.

Let's use '标注' here in line with the terminology provided [by the Chinese Translation Team of Rust](https://rustwiki.org/wiki/translate/english-chinese-glossary-of-rust/).

File: src/content/docs/zh-cn/features/commands.mdx [hunk 18]
```diff
@@ -0,0 +1,390 @@
...
+
+```js
+// 当使用 Tauri API 的 npm 包时
+import { invoke } from '@tauri-apps/api/core';
+
+// 使用 Tauri 全局脚本时（如果不使用 npm 包）
+// 确保在 `tauri.conf.json` 中将 `app.withGlobalTauri` 设置为 true。
+const invoke = window.__TAURI__.invoke;
+
+// Invoke the command
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
// 使用命令
```

File: src/content/docs/zh-cn/features/commands.mdx [hunk 19]
```diff
@@ -0,0 +1,390 @@
...
+
+参数应该作为一个带有 camelCase 键的 JSON 对象传递。
+
+```js
+invoke('my_custom_command', { invokeMessage: 'Hello!' });
+```
+
+参数可以是任何类型，只要它们实现了 [`serde::Deserialize`]。
+
+请注意，当在Rust中使用 snake_case 声明参数时，在 JavaScript 中参数会被转换为 camelCase。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
请注意，当在 Rust 中使用 snake_case 声明参数时，在 JavaScript 中参数会被转换为 camelCase。
```

File: src/content/docs/zh-cn/features/commands.mdx [hunk 20]
```diff
@@ -0,0 +1,390 @@
...
+#[tauri::command]
+fn my_custom_command() -> Result<(), Error> {
+	// 这将返回一个错误
+	std::fs::File::open("path/that/does/not/exist")?;
+	// 成功返回空
+	Ok(())
+}
+```
+
+自定义错误类型的优点是明确所有可能的错误，以便读者可以快速识别可能发生的错误。这为别人（和你自己）在以后审查和重构代码时节省了大量的时间。<br/>
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
自定义错误类型的优点是明列所有可能的错误，以便读者快速识别可能发生的错误。这为别人（和你自己）在以后审查和重构代码时节省了大量的时间。<br/>
```

File: src/content/docs/zh-cn/features/commands.mdx [hunk 21]
```diff
@@ -0,0 +1,390 @@
...
+它还让你可以完全控制错误类型被序列化的方式。在上面的例子中，我们简单地将错误消息作为字符串返回，但是你可以为每个错误分配一个类似于 C 的代码，这样你可以更容易地将它映射到一个看起来类似的 TypeScript 错误枚举。
+
+## 异步命令
+
+异步函数在 Tauri 中有利于执行繁重的工作，不会导致 UI 冻结或减慢。
+
+:::note
+
+异步命令使用 [`async_runtime::spawn`] 在单独的线程上执行。
+没有 _async_ 关键字的命令将在主线程上执行，除非定义了 _#[tauri::command(async)]_.
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
没有 _async_ 关键字的命令将在主线程上执行，除非定义了 _#[tauri::command(async)]_。
```

File: src/content/docs/zh-cn/features/commands.mdx [hunk 22]
```diff
@@ -0,0 +1,390 @@
...
+
+:::
+
+**如果你的命令需要异步运行，只需将其声明为 `async`。**
+
+:::caution
+
+使用 Tauri 创建异步函数时需要小心目前，你不能简单地在异步函数的签名中包含借用的参数。一些常见的类型示例是 `&str` 和 `State<'_, Data>`。
+这里跟踪了这个限制： https://github.com/tauri-apps/tauri/issues/2533 和解决方法如下所示。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
这里追踪了这个限制： https://github.com/tauri-apps/tauri/issues/2533 和解决方法如下所示。
```
'跟踪' is probably more like stalking, a bit weird using this word here.

File: src/content/docs/zh-cn/features/commands.mdx [hunk 23]
```diff
@@ -0,0 +1,390 @@
...
+
+:::
+
+在使用借用的类型时，必须进行额外的更改。这是你的两个主要选择。
+
+**选项 1**: 将类型（如 `&str`）转换为不允许借用的类似类型（如 `String`）。 这可能不适用于所有类型，例如 `State<'_, Data>`。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
**选项 1**：将类型（如 `&str`）转换为不允许借用的类似类型（如 `String`）。 这可能不适用于所有类型，例如 `State<'_, Data>`。
```

File: src/content/docs/zh-cn/features/commands.mdx [hunk 24]
```diff
@@ -0,0 +1,390 @@
...
+// 使用 String 而不是 &str 声明异步函数，因为 &str 是借用的，因此不受支持
+#[tauri::command]
+async fn my_custom_command(value: String) -> String {
+	// 调用另一个异步函数并等待它完成
+	some_async_function().await;
+	value
+}
+```
+
+**选项 2**: 将返回类型包装在一个 [`Result`] 中。这个实现起来有点难，但应该适用于所有类型。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/commands.mdx]:
```suggestion
**选项 2**：将返回类型包装在一个 [`Result`] 中。这个实现起来有点难，但应该适用于所有类型。
```

File: src/content/docs/zh-cn/features/file-system.mdx [hunk 25]
```diff
@@ -0,0 +1,491 @@
...
+       .expect("error while running tauri application");
+}
+```
+
+  </TabItem>
+</Tabs>
+
+## 安全
+
+这个模块阻止路径遍历，不允许绝对路径或父目录组件(即 `/usr/path/to/file` 或 `../path/to/file` 路径是不允许的)。使用这个 API 访问的路径必须相对于一个基本目录，因此如果你需要访问任意的文件系统路径，你必须在核心层上编写这样的逻辑。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/file-system.mdx]:
```suggestion
这个模块阻止路径遍历，不允许绝对路径或父目录组件（即 `/usr/path/to/file` 或 `../path/to/file` 路径是不允许的）。使用这个 API 访问的路径必须相对于一个基本目录，因此如果你需要访问任意的文件系统路径，你必须在核心层上编写这样的逻辑。
```

File: src/content/docs/zh-cn/features/single-instance.mdx [hunk 26]
```diff
@@ -0,0 +1,113 @@
...
+description: 确保一次只运行一个 Tauri 应用程序实例。
+---
+
+import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+<PluginLinks plugin="single-instance" showJsLinks={false} />
+
+使用单实例插件确保 tauri 应用程序在同一时间只运行单个实例。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/single-instance.mdx]:
```suggestion
使用单实例插件确保 Tauri 应用程序在同一时间只运行单个实例。
```

File: src/content/docs/zh-cn/features/single-instance.mdx [hunk 27]
```diff
@@ -0,0 +1,113 @@
...
+## 用法
+
+插件已经安装并初始化，应该可以立即正常运行。尽管如此，我们也可以使用 `init()` 方法来增强它的功能。
+
+插件的 `init()` 方法接受一个闭包，该闭包在新 App 实例启动时调用，但由插件关闭。
+这个闭包有三个参数：
+
+1. **`app` :** 应用程序的 [AppHandle](https://docs.rs/tauri/latest/tauri/struct.AppHandle.html) 。
+2. **`args` :** 用户初始化新实例时传递的参数列表。
+3. **`cwd` :** 当前工作目录表示启动新应用程序实例的目录。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/single-instance.mdx]:
```suggestion
1. **`app`**：应用程序的 [AppHandle](https://docs.rs/tauri/latest/tauri/struct.AppHandle.html) 。
2. **`args`**：用户初始化新实例时传递的参数列表。
3. **`cwd`**：当前工作目录表示启动新应用程序实例的目录。
```

File: src/content/docs/zh-cn/features/sql.mdx [hunk 28]
```diff
@@ -0,0 +1,221 @@
...
+			npm="npm add https://github.com/tauri-apps/tauri-plugin-sql#v2"
+			yarn="yarn add https://github.com/tauri-apps/tauri-plugin-sql#v2"
+			pnpm="pnpm add https://github.com/tauri-apps/tauri-plugin-sql#v2"
+		/>
+	</TabItem>
+</Tabs>
+
+## 用法
+
+首先，你需要在 Tauri 中注册插件:
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/sql.mdx]:
```suggestion
首先，你需要在 Tauri 中注册插件：
```

File: src/content/docs/zh-cn/features/store.mdx [hunk 29]
```diff
@@ -0,0 +1,183 @@
...
+
+// 设置一个值。
+await store.set('some-key', { value: 5 });
+
+// 获取一个值。
+const val = await store.get('some-key');
+console.log(val); // { value: 5 }
+
+// 您可以在进行更改后手动保存存储
+// 否则, 如上所述，它将在正常退出时保存。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/store.mdx]:
```suggestion
// 否则如上所述，它将在正常退出时保存。
```

File: src/content/docs/zh-cn/features/store.mdx [hunk 30]
```diff
@@ -0,0 +1,183 @@
...
+                // 注意，值必须是 serde_json::Value 的实例，
+                // 否则，它们将与 JavaScript 绑定不兼容。
+                store.insert("some-key".to_string(), json!({ "value": 5 }))?;
+
+                // 从 Store 中获取一个值。
+                let value = store.get("some-key").expect("Failed to get value from store");
+                println!("{}", value); // {"value":5}
+
+                // 您可以在进行更改后手动保存存储
+                // 否则, 如上所述，它将在正常退出时保存。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/store.mdx]:
```suggestion
                // 否则如上所述，它将在正常退出时保存。
```

File: src/content/docs/zh-cn/features/websocket.mdx [hunk 31]
```diff
@@ -0,0 +1,116 @@
+---
+title: Websocket
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/websocket.mdx]:
```suggestion
title: WebSocket
```

File: src/content/docs/zh-cn/features/window-customization.mdx [hunk 32]
```diff
@@ -0,0 +1,218 @@
...
+- [创建自定义标题栏](#创建自定义标题栏)
+- [（macOS） 具有自定义窗口背景颜色的透明标题栏](#macos-具有自定义窗口背景颜色的透明标题栏)
+
+### 创建自定义标题栏
+
+这些窗口特性的一个常用用途是创建自定义标题栏。这篇简短的教程将指导你完成这个过程。
+
+:::note
+对于 macOS，使用自定义标题栏也会失去系统提供的某些功能，比如[移动或对齐窗口](https：//support.apple.com/guide/mac-help/work-with-app-windows-mchlp2469/mac)。
+另一种自定义标题栏但保留原生功能的方法是使标题栏透明并设置窗口背景色，参见 [（macOS） 具有自定义窗口背景颜色的透明标题栏](#macos-具有自定义窗口背景颜色的透明标题栏) 的用法。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/window-customization.mdx]:
```suggestion
另一种自定义标题栏但保留原生功能的方法是使标题栏透明并设置窗口背景色，参见 [（macOS） 具有自定义窗口背景颜色的透明标题栏](#macos-具有自定义窗口背景颜色的透明标题栏)的用法。
```

File: src/content/docs/zh-cn/features/window-customization.mdx [hunk 33]
```diff
@@ -0,0 +1,218 @@
...
+		{
+			"decorations": false
+		}
+	]
+}
+```
+
+#### 权限
+
+在能力 （capability） 文件中添加窗口权限。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/window-customization.mdx]:
```suggestion
在能力（capability）文件中添加窗口权限。
```
Besides redundant space, I not quite sure if '能力' is the best word to use here. But let's maybe just roll with it first and patch it up later when someone had a better idea.

File: src/content/docs/zh-cn/features/websocket.mdx [hunk 34]
```diff
@@ -0,0 +1,116 @@
...
+
+- Windows
+- Linux
+- macOS
+
+## 设置
+
+_这个插件要求 Rust 版本至少是 **1.75**_
+
+首先安装 websocket 插件。
```
Comment (dklassic) [inline src/content/docs/zh-cn/features/websocket.mdx]:
```suggestion
首先安装 WebSocket 插件。
```
Starting to think maybe we should capitalize all English plugin names, but let's just change this one for now for consistency's sake.

**Review by dklassic — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/features/websocket.mdx (hunk 31 above)
Comment (GuoJikun) [reply src/content/docs/zh-cn/features/websocket.mdx:2]: websocket is a proper term!  Does s need to be capitalized as well?

File: src/content/docs/zh-cn/features/window-customization.mdx (hunk 33 above)
Comment (GuoJikun) [reply src/content/docs/zh-cn/features/window-customization.mdx]: agree

**Review by dklassic — APPROVED** [review]

**Comment by GuoJikun** [comment]
Excuse me, is the information on this page accurate? It doesn't look right.  @dklassic 
image

**Comment by dklassic** [comment]
> Excuse me, is the information on this page accurate? It doesn't look right. @dklassic

Actually I have no idea on how that page works as I'm not from doc team, just helping out here. You can open an issue for this or maybe seek some answers on Discord.

**Pull request merged**